_maps/__MAP_DEFINES.dm

```diff
@@ -31,7 +31,7 @@
 	#define CENTCOMM "CentComm"
 	#define TELECOMMS "Telecomms Satellite"
 	#define DERELICT "Derelicted Station"
-	#define MINING "Mining Asteroid"
+	#define MINING "Lavaland"
```

Comment:
The mining asteroid's still around and (when I last tested it) fully functional. I know some people have concerns if Lavaland will suit Paradise, so one option is to add the asteroid on a separate z-level and give miners a choice. Not sure how well that'll work though.

Reply:
Or maybe an option in the mining shuttle console to determine which it lands at :thinking:

Reply:
That shouldn't be too hard to implement I think, shuttle consoles already support multiple destinations. Or we could make a smaller mining asteroid to safely get some minerals, instead of the fully sized one? Something for later, I assume.